admin-dev/themes/new-theme/js/pages/order/create/cart-editor.js

```diff
@@ -68,12 +68,17 @@ export default class CartEditor {
    * Changes cart free shipping value
    *
    * @param {Number} cartId
-   * @param {Boolean} value
    */
-  setFreeShipping(cartId, value) {
-    $.post(this.router.generate('admin_carts_set_free_shipping', {cartId}), {
-      freeShipping: value,
-    }).then(cartInfo => EventEmitter.emit(eventMap.cartFreeShippingSet, cartInfo))
+  updateDeliveryOptions(cartId) {
+    const freeShippingEnabled = $(createOrderMap.freeShippingSwitch)[1].checked;
+    const isAGiftEnabled = $(createOrderMap.isAGiftSwitch)[1].checked;
```

Comment:
You can select the value of the checked one with
```suggestion
    const isAGiftEnabled = $(createOrderMap.isAGiftSwitch).val() === '1';
```